pkgs/development/haskell-modules/configuration-hackage2nix.yaml

```diff
@@ -2498,6 +2498,7 @@ default-package-overrides:
 
 extra-packages:
   - aeson < 0.8                         # newer versions don't work with GHC 7.6.x or earlier
+  - ansi-terminal == 0.10.3             # required by cabal-plan, and policeman in ghc-8.8.x
```

Comment:
I added this so that `haskell-updates` won't get into a non-evaluatable state because of `cabal-plan` and `policeman`'s use of `ansi-terminal`.